src/main/scala/org/holmesprocessing/totem/services/pemeta/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:alpine
+
+# Create folder
+RUN mkdir -p /service
+WORKDIR /service
+
+# Get Golang and PEV dependencies
+RUN apk add --no-cache \
+        git \
+        && go get github.com/julienschmidt/httprouter \
+        && git clone -b testing https://github.com/boddumanohar/libpe.git \
```

Comment:
That is really great to hear that you are contributing to the other project as well. Good job and good luck with the pull request to pev!